preload

```diff
@@ -37,25 +37,18 @@ function error {
   exit 1
 }
 
-#yad or xlunch format
+case "$1" in
+source)  return 0 ;;
```

Comment:
The `preload` script is not to be sourced. Why has this input argument been added?